From 1ec77071c504cdf962b1d642ba1588df7a046774 Mon Sep 17 00:00:00 2001
From: Leo Singer <leo.singer@ligo.org>
Date: Sat, 17 Apr 2021 09:56:37 -0400
Subject: [PATCH] Add missing (void) to prototype

LALSuite builds with FrameL are failing because LALSuite is built
with the `-Wstrict-prototypes` warning enabled, which requires
prototypes for functions to take no arguments to be declared with
the argument list `(void)`.

Once this is merged and in a release, we will be able to address
lscsoft/lalsuite#437.
---
 src/FrameL.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/FrameL.h b/src/FrameL.h
index d8c097e..4a31b65 100644
--- a/src/FrameL.h
+++ b/src/FrameL.h
@@ -632,7 +632,7 @@ FrVect *FrFileIGetVType(FrFile *iFile, const char *name, double tStart, double l
 double FrFileIGetNextEventTime(FrFile *iFile, const char *name, double tStart,
                                   double amplitudeMin);
 int     FrFileIGoToNextRecord(FrFile *iFile);
-FrFile* FrFileIInMemoryNew();
+FrFile* FrFileIInMemoryNew(void);
 double  FrFileIInMemoryAddBuffer(FrFile* file, char* buf, int size, double tSpan);
 FrFile *FrFileINew  (const char *fileName);
 FrFile *FrFileINewFd(FrIO *frfd);
-- 
2.31.1

